relay-server/src/actors/connector.rs

```diff
@@ -43,6 +46,24 @@ impl MeteredConnector {
     pub fn start(config: Arc<Config>) -> Addr<Self> {
         let metered_connector = Self.start();
 
+        // Relay communicates with a single upstream that is supposed to be known ahead of time. If
+        // that upstream cannot be resolved, this is usually due to one of few reasons:
+        //
+        //  1. A typo in the configuration. This is usually caught during development time and then
+        //     fixed by the author.
+        //  2. A service like load balancer or docker container starting up with delay. In such a
+        //     case, Relay is expected to connect as soon as the service becomes available.
+        //  3. Misconfiguration in the network, most likely DNS server.
+        //
+        // By default, Relay detects these issues during startup time and employs exponential
+        // backoff to reconnect. Since outgoing traffic is of low volume at this point, the TTL for
+        // negative DNS caches can be set to a few seconds.
+        let mut opts = ResolverOpts::default();
+        opts.negative_max_ttl = Some(Duration::from_secs(3));
```

Comment:
What's the default value?

Reply:
I'm not 100% sure, but the default value seems to be 1 day.